addons/rose/addon/components/rose/nav/sidebar/index.hbs

```diff
@@ -12,9 +12,11 @@
 >
 
   {{#if @title}}
-    <span class='rose-nav-title' id='title-{{this.id}}'>
+    {{! Updated span however this is hidden by default in the desktop,
+    once side nav refactor is done this will ultimately be removed }}
```

Comment:
Once the side nav in the desktop client is updated to use Hds components, this will be removed but I felt it wise to update it regardless.